lib/challenge.js

```diff
@@ -58,9 +97,25 @@ var some_pet;
 //    the values of `name`, `species`, `breed`, and `noise`. The constructor function should also give each pet a method called `makeNoise`,
 //    which behaves as described in Q3.
 
-var Pet;
 // Answer Starts Here
 
+var Pet = function(name, species, breed, noise){
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.noise = noise;
+  // option 1
+  this.makeNoise = function(){
+    return this.noise;
+  }
+}
+
+// OR option 2
```

Comment:
Not 'or' - you need to do _both_. Attributes get defined in the Constructor, methods get defined on the Constructor's prototype.